src/pretix/base/exporters/orderlist.py

```diff
@@ -424,13 +424,13 @@ def iterate_fees(self, form_data: dict):
         ).values(
             'm'
         ).order_by()
-        qs = OrderFee.objects.filter(
+        qs = OrderFee.all.filter(
```

Comment:
Thanks for this! I wondered why this change, but now learned `objects` is a Manager with a filter removing canceled fees. Good to know!